src/components/organisms/CategoryList/index.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useRef, useEffect } from 'react';
+import Styled from 'styled-components';
+import CategoryButton from 'components/molecules/CategoryButton';
+import moreIcon from 'assets/images/moreIcon.svg';
+import nextIcon from 'assets/images/nextIcon.svg';
+import moreClickedIcon from 'assets/images/more_filled.svg';
+interface IProps {
+  reRender?: (interest: string) => void;
+}
+function CategoryList({ reRender }: IProps): React.ReactElement {
+  const totalSlide = 6;
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef<HTMLInputElement>(null);
+  const [openMore, setOpenMore] = useState(false);
+  const [moreClicked, setMoreClicked] = useState(false);
+  const [interest, setInterest] = useState('');
+
+  const onClickOpenMore = () => {
+    setOpenMore(!openMore);
+    setMoreClicked(!moreClicked);
+  };
+  const onClickInterest = (newInterest: any) => {
+    setInterest(newInterest);
+    reRender && reRender(interest);
+  };
+
+  const nextSlide = () => {
+    console.log(currentSlide);
+    if (currentSlide >= totalSlide) {
+      setCurrentSlide(0);
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
+  };
+  useEffect(() => {
+    if (slideRef?.current) {
+      slideRef.current.style.transition = 'all 1s ease-in-out';
+      slideRef.current.style.transform = `translateX(-${currentSlide * 15}%)`;
+    }
+  }, [currentSlide]);
+
+  return (
+    <SCategoryList>
+      <div className="main">
+        <CategoryButton className="fix" tag="전체"></CategoryButton>
+        <div className="hashtag__container">
+          <div className="hashtags" ref={slideRef}>
+            {hashtagList.map((tag, index) => (
```

Comment:
해시태그 리스트가 없을 때를 대비해서
```suggestion
            {hashtagList?.map((tag, index) => (
```
하면 에러가 안나겠죠??